tests/behaviour/contracts/cairo_stubs/basic.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.8.10;
+
+contract WARP {
+  /// warp-cairo
+  /// func __warp_usrfn0_stub() -> (a: felt):
```

Comment:
My only concern is how the user knows what the mangled name is. Would we expect the user to transpile the contract to fetch it?

Reply:
No, there is the INTERNALFUNC macro, I just hadn't added it when I wrote that example. I'll update it to use the macro